addons/html_builder/static/src/plugins/button_option.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="html_builder.ButtonOption">
+    <BuilderRow label.translate="Style" level="1">
+        <BuilderSelect>
+            <BuilderSelectItem classAction="''">Link</BuilderSelectItem>
+            <!-- Todo: when merging, change here + s_button.xml to btn-fill-primary -->
```

Comment:
Could you explain this TODO ?

Reply:
It's a spec change in editor that the button is `fill` by default, but I will re-check with lebl about it :pray: